test/fixtures/integration/help.expected.log

```diff
@@ -4,10 +4,12 @@ Picked up _JAVA_OPTIONS: -Dfile.encoding=UTF-8
 Commands:
   puzzle list           List available puzzle types
   puzzle gen <type>     Generate a new puzzle
+  puzzle show <code>    Print a puzzle
   puzzle solve <code>   Print the solution to a puzzle
 
 Options:
   -d,--difficulty <num>      Change puzzle difficulty from default
+                             Use -1 to see range and minimum for credit
```

Comment:
Oh, that's a good idea! Let's make it `-d help` or similar; special-casing -1 feels just a little clumsy.